astropy/cosmology/core.py

```diff
@@ -28,7 +27,7 @@
            "Flatw0waCDM", "w0waCDM", "wpwaCDM", "w0wzCDM",
            "default_cosmology"] + parameters.available
 
-__doctest_requires__ = {'*': ['scipy.integrate', 'scipy.special']}
+__doctest_requires__ = {'*': ['scipy']}
```

Comment:
FWIW, I think `__doctest_requires__ = {'*': ['scipy']}` actually makes more sense, as it is not a namespace package, so if you can import `scipy`, chances are you can import the subpackage as well? 🤷‍♀

Reply:
the argument for supporting the subpackage version as well is that it can be a new subpackage, e.g. importing astropy doesn't mean that `astropy.timeseries` would be available for those tests. Anyway, it's theoretical now as all these scipy functionalities are old enough, and https://github.com/astropy/pytest-doctestplus/issues/67 has been opened upstream.